src/foam/nanos/notification/email/EmailMessage.js

```diff
@@ -105,9 +106,9 @@ foam.CLASS({
       tableWidth: 100
     },
     {
-      class: 'String',
-      name: 'templateName',
-      value: null
+      class: 'Map',
```

Comment:
Do you not need templateName?

Reply:
One of the key/value pairs in templateArguments is [emailTemplate, emailTemplateName].